src-self-hosted/link.zig

```diff
@@ -2204,7 +2204,7 @@ pub const File = struct {
                     try dbg_info_buffer.writer().print("{}\x00", .{ty});
                 },
                 else => {
-                    log.err(.compiler, "TODO implement .debug_info for type '{}'", .{ty});
+                    std.log.scoped(.compiler).err("TODO implement .debug_info for type '{}'", .{ty});
```

Comment:
This is the only log statement in link.zig that uses .compiler as its scoped instead of .link; I assume that's on purpose?

Reply:
yeah that was on purpose, thanks for checking though!